modules/widgets/search.php

```diff
@@ -269,6 +269,19 @@ public function widget( $args, $instance ) {
 
 		$display_filters = false;
 
+		if ( Jetpack::is_development_mode() ) {
```

Comment:
This seems to be duplicating what's already below.

Reply:
Nice catch. Fixed